.azure-pipelines.yml

```diff
@@ -13,21 +13,29 @@ jobs:
     name: Windows_NT
     displayName: Innerloop Windows_NT
     targetOS: Windows_NT
-    pool:
-      name: Hosted VS2017
 
-- template: /pipelines/corefxlab-base.yml
-  parameters:
-    name: MacOS
-    displayName: Innerloop OSX
-    targetOS: OSX
-    pool:
-      name: Hosted macOS
+    ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
nit: Is there a way to save such a condition "locally" in yml and re-use it/!it (not it) to avoid duplication/typos between the negation?

Reply:
I heard they were going to add support through importing a file that defines constants, but I haven't heard of that support being added, that is why we need to pass it down as a parameter to the template, that is the only way I know of caching the value.

cc: @chcosta do you know if this is now supported?